### PR TITLE
localvolumeprovisioner: single storageclass with flag for default

### PR DESCRIPTION
**Pull request by gpaul**
Modify the localvolumeprovisioner to specify a hardcoded StorageClass.
The StorageClass has a single configuration option,
'storageclassdefault: true|false' that specifies whether or not it
should be annotated as the default StorageClass.

### CODE REVIEW

**Comment by joejulian** [comment]
The chart is now missing a default values.yaml

**Comment by gpaul** [comment]
@joejulian good point - PTAL

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,9 @@ STABLE_TARGETS = $(shell hack/chart_destination.sh $(STABLE_CHARTS))
 STAGING_CHARTS = $(wildcard staging/*/Chart.yaml)
 STAGING_TARGETS = $(shell hack/chart_destination.sh $(STAGING_CHARTS))
 
+GIT_REMOTE_URL := $(shell git remote get-url origin)
+GITHUB_USER := $(shell git remote get-url origin | cut -d':' -f2 | cut -d'/' -f1)
```
Comment (hectorj2f) [inline Makefile]: This line return an empty space in my case. Please, fix it and open a different PR cause the changes here and in the title of the PR target different goals.

Comment (gpaul) [reply Makefile]: :thinking: Thanks for checking - I've moved that commit to https://github.com/mesosphere/charts/pull/10 and will drop it from this PR.

File: stable/localvolumeprovisioner/templates/NOTES.txt [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
 
-{{- range .Values.storageclasses }}
-/mnt/{{ .name }}
-{{- end }}
+/mnt/disks/
 
-The directory names match their corresponding storage classes.
-By mounting volumes into these directories, they will be made available
-as persistent volume in their respective storage class.
+Volumes that are mounted into this directory show up as PersistentVolumes. Their
```
Comment (nfnt) [inline stable/localvolumeprovisioner/templates/NOTES.txt]:
```suggestion
Volumes that are mounted into this directory show up as persistent volumes. Their
```

File: stable/localvolumeprovisioner/templates/NOTES.txt [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
+`storageClassName` is set to `"localvolumeprovisionerstorageclass"`.
```
Comment (nfnt) [inline stable/localvolumeprovisioner/templates/NOTES.txt]:
```suggestion
`storageClassName` is set to `"localvolumeprovisioner"`.
```

File: stable/localvolumeprovisioner/templates/deployment.yaml [hunk 4]
```diff
@@ -45,16 +45,14 @@ metadata:
   namespace: kubeaddons
 data:
   storageClassMap: |
-    {{- range .Values.storageclasses }}
-    {{ .name }}:
-       hostDir: /mnt/{{ .name }}
-       mountDir: /mnt/{{ .name }}
+    localvolumeprovisionerstorageclass:
```
Comment (nfnt) [inline stable/localvolumeprovisioner/templates/deployment.yaml]:
```suggestion
    localvolumeprovisioner:
```

File: stable/localvolumeprovisioner/templates/storageclass.yaml [hunk 5]
```diff
@@ -1,16 +1,11 @@
-{{- range .Values.storageclasses }}
 ---
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: {{ .name }}
-  {{- if .annotations }}
+  name: localvolumeprovisionerstorageclass
```
Comment (nfnt) [inline stable/localvolumeprovisioner/templates/storageclass.yaml]:
```suggestion
  name: localvolumeprovisioner
```

**Comment by gpaul** [comment]
Thanks @nfnt that's better. Comments addressed PTAL

**Pull request merged**